src/GovUk.Education.ExploreEducationStatistics.Common/Converters/TimeIdentifierCategoryJsonConverter.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Converters
+{
+    public class TimeIdentifierCategoryJsonConverter : JsonConverter<TimeIdentifierCategory?>
+    {
+        public override void WriteJson(JsonWriter writer, TimeIdentifierCategory? value, JsonSerializer serializer)
+        {
+            writer.WriteStartObject();
```

Comment:
This seems quite manual.  I'm totally guessing that you've ruled out that this can't be inferred automatically?

Reply:
Its cause we want to split an enum out into bits which is kinda a manual process - can't see any obvious other way to do it. Well we could put it in a view model - but that would potentially mean a new view model for all instances where we have one of these enums embedded.